release/src/main/scripts/build_release_candidate.sh

```diff
@@ -236,6 +236,9 @@ if [[ $confirmation = "y" ]]; then
     ./gradlew ":runners:flink:${ver}:job-server-container:dockerPush" -Pdocker-tag="${RELEASE}_rc${RC_NUM}"
   done
 
+  echo '-------------Generating and Pushing Spark job server image-------------'
+  ./gradlew "./gradlew :runners:spark:job-server:container:dockerPush" -Pdocker-tag="${RELEASE}_rc${RC_NUM}"
```

Comment:
Don't we need to specify spark version here?

Reply:
No, the Spark runner only supports one Spark version per Beam release (currently [2.4.5](https://github.com/apache/beam/blob/800754399baa939232b9863b5104111c638e88aa/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L399)), unlike Flink, which supports the latest three Flink versions.

Reply:
How about in future? In addition, I think users should know which spark version is supported even if only one version is supported.

Reply:
Minor clarification to the above: Beam Spark runner supports Spark's latest _feature_ release, i.e. `2.4.*`.

I think the plan is to create a separate module when moving to Spark 3 ([BEAM-9282](https://issues.apache.org/jira/browse/BEAM-9282)). @iemejia knows more

Reply:
Then let’s add it to documentation some where. If I am a spark user, I would really wonder which version is being supported.

Reply:
It is documented on the Spark runner page: https://beam.apache.org/documentation/runners/spark/

Reply:
Just noticed that `./gradlew` is repeated here as well.

Reply:
😰 fixed